fury/actors/peak.py

```diff
@@ -40,11 +40,15 @@ class PeakActor(Actor):
         :func:`fury.actor.colormap_lookup_table`.
     linewidth : float, optional
         Line thickness. Default is 1.
+    symmetric: bool, optional
+        If True, peaks are drawn for both peaks_dirs and -peaks_dirs. Else,
+        peaks are only drawn for directions given by peaks_dirs. Default is
+        True.
 
     """
 
     def __init__(self, directions, indices, values=None, affine=None,
-                 colors=None, lookup_colormap=None, linewidth=1):
+                 colors=None, lookup_colormap=None, linewidth=1, symmetric=True):
```

Comment:
Thanks for all the updates.

I just have noticed that this line is rising: `PEP 8: E501 line too long (81 > 79 characters)`